alicloud/resource_alicloud_cs_kubernetes.go

```diff
@@ -566,19 +636,68 @@ func resourceAlicloudCSKubernetesDelete(d *schema.ResourceData, meta interface{}
 	})
 }
 
-func buildKunernetesArgs(d *schema.ResourceData, meta interface{}) (*cs.KubernetesCreationArgs, error) {
+func isMultiAZClusterAndCheck(d *schema.ResourceData) (bool, error) {
+	masterInstanceTypes := expandStringList(d.Get("master_instance_types").([]interface{}))
+	workerInstanceTypes := expandStringList(d.Get("worker_instance_types").([]interface{}))
+	vswitchIDs := expandStringList(d.Get("vswitch_ids").([]interface{}))
+	workerNumbers := expandIntList(d.Get("worker_numbers").([]interface{}))
+
+	if len(masterInstanceTypes) != len(workerInstanceTypes) {
```

Comment:
@xiaozhu36 If lengths of masterInstanceTypes and workerInstanceTypes are both 3, length of vswitch_ids should be three as well.
Otherwise, the cluster is a singleaz cluster, length of vswitch_ids can be 0 or 1.